Gemfile

```diff
@@ -5,3 +5,5 @@ source "https://rubygems.org"
 # gem "rails"
 ruby '3.1.0'
 gem 'rspec'
+gem 'rubocop', require: false
+gem 'simplecov', require: false, group: :test
```

Comment:
You are able to organize your gemfile groups like this as well:

```rb
group :test do
  gem 'simplecov', require: false
end
``` 
You can check this link for a little more information about that: [Bundler and Gemfile best practices](https://depfu.com/blog/2017/01/18/bundler-and-gemfile-best-practices#use-gemfile-groups)

What do you think?